dispatch/route.go

```diff
@@ -180,6 +180,30 @@ func (r *Route) Key() string {
 	return b.String()
 }
 
+// ID returns a key for the route. It should uniquely identify the route in general,
+// it is different than Key() as it adds the route's position on its parent's children.
```

Comment:
(nit) not sure if we need to go into the implementation details here. 
```suggestion
// ID returns a unique identifier for the route, unlike Key().

Reply:
fixed